### ISSUE TITLE
The error message shows incomplete when typing a function name that started or ended of a hyphen

### ISSUE THREAD

**Issue by v-mengwe** [microsoft/vscode-azurefunctions#2927]
**OS:** Win10/Mac
**Build Version:** 20210819.3
**Regression:** Regression

**Repro Steps:**

1. F1 ->  Azure Functions: Create Function App in Azure -> Select a subscription -> Type a function name that started or ended of a hyphen.
2. Check whether the error message shows complete.

**Expect:**
The error message shows complete.
![image](https://user-images.githubusercontent.com/62053665/130214838-e5d59597-b25c-419b-931e-4d97256683b2.png)

**Actual:**
The error message shows incomplete.
![image](https://user-images.githubusercontent.com/62053665/130214781-fc22c80b-075f-4392-acb0-90b00d5fa74f.png)

**More Info:**

1. This issue doesn't reproduce on the latest release build.
2. Mouse hover can't fix this issue.

**Comment by ejizba** [comment]
This is likely something caused by VS Code, but it's interesting that it repros in our latest release build and not on main. We can investigate

**Comment by ejizba** [comment]
Yeah this is reproing for me on VS Code insiders regardless of our extension's build, but not on VS Code stable.

**Comment by alexweininger** [comment]
Tracking: https://github.com/microsoft/vscode/issues/131460
